apps/www/components/Article/DynamicComponents.tsx

```diff
@@ -0,0 +1,177 @@
+import { Loader } from '@project-r/styleguide'
+
+import dynamic from 'next/dynamic'
+
+// CAMPAIGN MODE
+// import { TrialPaynote } from '@app/app/(campaign)/components/trial-paynote'
```

Comment:
I guess we could also remove these campaign paynotes too

Reply:
ah yes!